package.json

```diff
@@ -22,29 +22,45 @@
       "git add"
     ]
   },
+  "dependencies": {
+    "react": "^16.8.6",
```

Comment:
noticed these were in `devDependencies` instead of `dependencies` and was throwing error

Reply:
Ah actually these can't be in `dependencies` (react, react-dom, styled-components, etc` because otherwise projects that install `@stellar/elements` will end up with 2 copies of React loaded. Needs to stay in `devDependencies` + `peerDependencies`.